Common/DTO/User.cs

```diff
@@ -2,6 +2,11 @@
 {
     public class User
     {
+        public User (string email, string password)
+        {
+            Email = email;
+            Password = password;
+        }

```

Comment:
may be need empity constructor too? I think, this need to correct work <Editform> tag in razor

Reply:
I can add it. But I think it had to be added by people who need this. Are your command need this right now? I hadn't worked with razor pages yet.
Also, there was an idea to create class UserEmailPassword (don't know how to name it) With only 2 properties: email and password, obviously. It could be useful to send less data

Reply:
may be create email_id_user and id_login_user and oter? Why we need create another DTO? Ok, how we can set user ID and login? it is readonly

Reply:
Okay, probably just id, email, password. Look, User will have more properties like LastName, FirstName, Birthdate, etc. And I think it isn't good to move all structure when we just want to register or authorize.

Reply:
not actual task